src/index.js

```diff
@@ -0,0 +1,151 @@
+
+const API = "http://127.0.0.1:5000";
+const locationURL = "/location";
+const weatherURL = "/weather";
+
+//look more into State 
+const state = {
+    city: 'Seattle',
+    liveTemp : 68,
+    lon: -122.3300624,
+    lat: 47.6038321
+};
+
+const changeTempAndLand = () => {
+    let temp = state.liveTemp;
+    let color = 'yellow';
+    let land = "🌾🌾_🍃_🪨_🛤_🌾🌾_🍃";
+    if (temp >= 80) {
+        color = 'hot';
+        land = "🌵_🐍🌵🦂_🌵🌵__🐍🏜🦂";
+    } else if (temp >= 70) {
+        color = 'warm';
+        land = "🌸🌿🌼🌱_🌷🌻🌿_🌱🌻🌷";
+    } else if (temp >= 60) {
+        color = 'moderate';
+        land = "🌾🌾_🍃_🪨_🛤_🌾🌾_🍃🌳";
+    } else if (temp >= 50) {
+        color = 'cool';
+        land = "🌲🍃🌲🍂🌲🍁🌳🍃🌲🍂🌲🍃";
+    } else {
+        color = 'cold';
+        land = "🏔️⛄️🎄☃️❄️🏔️❄️☃️❄️🎄🏔️";
+    }
+    const temperature = document.getElementById('live_temp');
+    temperature.textContent = state.liveTemp;
+    temperature.className = color;
+    const landscape = document.getElementById('garden_land');
+    landscape.textContent = land;
+};
+
+const tempIncrease = () => {
+    state.liveTemp += 1;
+};
+
+const tempDecrease = () => {
+    state.liveTemp -= 1;
+};
+changeTempAndLand();
+
+const displayCityInput = () => {
+    let cityInput = document.getElementById("city_input").value;
+    let cityDisplay = document.querySelector("h2");
+    cityDisplay.textContent = `For the city of  ${cityInput}`;
+}
+
+const getCityInput = () => {
+    let cityInput = document.getElementById("city_input").value;
+    return cityInput;
+}
+const tempConverter = (kelvin) => {
+    return (kelvin - 273.15) * 1.8 + 32;
+
+}
+const getWeatherInfo = (lat, lon) => {
+    const weatherRoute = API + weatherURL;
+    axios
+        .get(weatherRoute, { params: { lat : `${lat}`, lon : `${lon}`, format: "json", units: "imperial"} })
+        .then((result) => {
+            tempInKel = result.data["main"]["temp"];
+            let resultFahr = Math.round(tempConverter(tempInKel));
+            let cityTemp = document.getElementById("live_temp");
+            cityTemp.innerHTML = resultFahr;
+            state.liveTemp = resultFahr;
+            changeTempAndLand();
+        })
+};
+const getCityLiveWeather = () => {
+    const cityName = document.getElementById("city_input").value;
+    const locationRoute = API + locationURL;
+    let tempInKel = 0;
+    axios
+        .get(locationRoute, { params: { q : `${cityName}`, format: "json"} })
```

Comment:
The only parameter accepted by the proxy server is `q`. Adding `format` doesn't do anything.